src/components/SyntaxHighlighter/YDBSyntaxHighlighter.tsx

```diff
@@ -0,0 +1,79 @@
+import {ClipboardButton} from '@gravity-ui/uikit';
+import {PrismLight as ReactSyntaxHighlighter} from 'react-syntax-highlighter';
+import bash from 'react-syntax-highlighter/dist/esm/languages/prism/bash';
+import cpp from 'react-syntax-highlighter/dist/esm/languages/prism/cpp';
+import csharp from 'react-syntax-highlighter/dist/esm/languages/prism/csharp';
+import go from 'react-syntax-highlighter/dist/esm/languages/prism/go';
+import java from 'react-syntax-highlighter/dist/esm/languages/prism/java';
+import javascript from 'react-syntax-highlighter/dist/esm/languages/prism/javascript';
+import php from 'react-syntax-highlighter/dist/esm/languages/prism/php';
+import python from 'react-syntax-highlighter/dist/esm/languages/prism/python';
```

Comment:
I don't think we need these languages =D

Are they necessary for syntax-highlighter to work?

Reply:
Ah well I see - there is some db connection code writeen with them